apps/web/src/api/api.client.ts

```diff
@@ -2,9 +2,9 @@ import axios from 'axios';
 import { API_ROOT } from '../config';
 
 export const api = {
-  get(url: string) {
+  get(url: string, absoluteUrl = false) {
```

Comment:
please make this second arg as options object like: `{ absoluteUrl }: { absoluteUrl?: boolean } = {}`, then it will be more readable from the plate where the `get` is used

Reply:
Good idea!!